name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeed.java

```diff
@@ -458,9 +458,12 @@ private LatestSecurityPrice extractPrice(Element row, List<Spec> specs) throws P
 
         LatestSecurityPrice price = new LatestSecurityPrice();
 
-        for (Spec spec : specs)
-            spec.column.setValue(cells.get(spec.index), price);
-
+        for (Spec spec : specs){
+            if(!"-".equalsIgnoreCase(cells.get(spec.index).text().trim())){
```

Comment:
hmm das könnte wohl in der Tat passieren, da hab ich auf die Schnelle nicht dran gedacht. Für mein verlinktes Beispiel, d.h. das auslesen von Onvista, passt es so, da es in dem Fall wohl immer nur einen "Schluss"-Kurs gibt. Die Specs (Spalten) sind ja alle da, nur die Preise an sich nicht vollständig für alle Spalten. Mit meiner Änderung wird das Werfen der entspr. Exceptions natürlich unterdrückt (unschön), ohne selbige kann man aber momentan in dem Fall gar nichts importieren und die Buttons weiter/Fertigstellen im Dialog sind ausgegraut.

Hier mal der Stacktrace (ohne Änderung):
Tue Jun 14 00:01:00 CEST 2016
Unparseable number: "-"

java.text.ParseException: Unparseable number: "-"
	at java.text.NumberFormat.parse(NumberFormat.java:385)
	at name.abuchen.portfolio.online.impl.HTMLTableQuoteFeed$Column.asQuote(HTMLTableQuoteFeed.java:90)
	at name.abuchen.portfolio.online.impl.HTMLTableQuoteFeed$HighColumn.setValue(HTMLTableQuoteFeed.java:161)
	at name.abuchen.portfolio.online.impl.HTMLTableQuoteFeed.extractPrice(HTMLTableQuoteFeed.java:459)
	at name.abuchen.portfolio.online.impl.HTMLTableQuoteFeed.parse(HTMLTableQuoteFeed.java:347)
	at name.abuchen.portfolio.online.impl.HTMLTableQuoteFeed.parseFromHTML(HTMLTableQuoteFeed.java:321)
	at name.abuchen.portfolio.online.impl.HTMLTableQuoteFeed.getHistoricalQuotes(HTMLTableQuoteFeed.java:284)
	at name.abuchen.portfolio.ui.wizards.datatransfer.ReviewImportedQuotesPage.beforePage(ReviewImportedQuotesPage.java:42)
	at name.abuchen.portfolio.ui.wizards.AbstractWizardPage$1.pageChanged(AbstractWizardPage.java:37)